04.ls/ls.rb

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: true
+
+NUM_COLUMNS = 3
+DEFAULT_COLUMN_SPACE = 5
+LAST_COLUMN_SPASE_NONE = 2
+HALF_WIDTH_BYTE = 1
+HALF_WIDTH_SPACE = 1
+FULL_WIDTH_SPACE = 2
+
+file_names =  Dir.glob('*')
+
+total_files = file_names.length
+row_count = total_files / NUM_COLUMNS
+
+def max_byte_size(file_names)
+  file_names.map(&:bytesize).max
+end
+
+def column_space(file_name, file_name_max_byte, loop_count)
+  column_spacing = ''
+  if loop_count <= NUM_COLUMNS - LAST_COLUMN_SPASE_NONE
+    padding_size = file_name_max_byte - text_byte_size(file_name) + DEFAULT_COLUMN_SPACE
+    padding_size.times do
+      column_spacing += ' '
+    end
+  end
+  column_spacing
```

Comment:
特に `column_spacing` に破壊的に代入する意味がないと思うので、

```suggestion
  if loop_count <= NUM_COLUMNS - LAST_COLUMN_SPASE_NONE
    padding_size = file_name_max_byte - text_byte_size(file_name) + DEFAULT_COLUMN_SPACE
    ' ' * padding_size
  else
    ''
  end
```

とするほうが見やすいかなと思います。